dspace-api/src/main/resources/spring/spring-dspace-addon-import-services.xml

```diff
@@ -63,6 +75,42 @@
           </property>
     </bean>
 
+    <bean id="CsvImportService"
+          class="org.dspace.importer.external.csv.service.CharacterSeparatedImportMetadataSourceServiceImpl" scope="singleton">
+          <property name="metadataFieldMap" ref="csvMetadataFieldMap"></property>
+          <property name="skipLines" value="1" />
+          <property name="supportedExtensions">
+               <list>
+                     <value>csv</value>
+               </list>
+          </property>
+    </bean>
+
+    <bean id="TsvImportService"
+          class="org.dspace.importer.external.csv.service.CharacterSeparatedImportMetadataSourceServiceImpl" scope="singleton">
```

Comment:
Scope singleton is the default, you can leave out the initial scope declaration.